openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMUtils.java

```diff
@@ -82,22 +85,33 @@ static CpuArchitecture getCpuArchitecture(final String cpuArchName) {
         }
     }
 
+    static LibcImplementation getLibcImplementation(final String libcImpl) {
+        if (libcImpl == null || libcImpl.isEmpty()) return LibcImplementation.GLIBC;
```

Comment:
Please add a log at debug level stating that FDZ_OPENML_JAVA_LIBC isn't set, hence we are assuming glibc as libc implementation.

Reply:
Added in the latest commit.